index.bs

```diff
@@ -1007,6 +1007,62 @@ default-reader-asynciterator-prototype-internal-slots">Asynchronous iteration</h
  1. Return [=a promise resolved with=] undefined.
 </div>
 
+<h3 id="generic-reader-mixin">The {{ReadableStreamGenericReader}} interface mixin</h3>
```

Comment:
Let's just say "mixin" instead of "interface mixin". And below, instead of "Interface definition", "Mixin definition".